tests/network/bgp/evpn/conftest.py

```
@@ -0,0 +1,185 @@
+    with create_evpn_frr_configuration(
+        name="frr-configuration-evpn",
+        frr_pod_ipv4=frr_external_pod.ipv4,
+        client=admin_client,
+        label=EVPN_ADVERTISE_LABEL,
+    ):
+        yield
+
+
+@pytest.fixture(scope="module")
+def evpn_setup_ready(
+    ovn_local_gateway_mode: None,
```

Ruff (0.15.11)

[warning] Unused function argument: `ovn_local_gateway_mode`

(ARG001)